src/index.html

```diff
@@ -7,12 +7,62 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/index.scss"
```

Comment:
The stylesheet link references a SCSS file (`styles/index.scss`). Browsers do not directly understand SCSS files; they need to be compiled into CSS first. Ensure that the SCSS file is compiled to a CSS file and update the `href` attribute to point to the compiled CSS file.